src/main/kotlin/uk/gov/gdx/datashare/services/ScheduledJobService.kt

```diff
@@ -20,8 +20,8 @@ class ScheduledJobService(
   @SchedulerLock(name = "countUnconsumedEvents", lockAtMostFor = "3m", lockAtLeastFor = "3m")
   fun countUnconsumedEvents() {
     LockAssert.assertLocked()
-    EventDataService.log.debug("Looking for unconsumed events")
-    val unconsumedEventsCount = eventDataRepository.countByDeletedAtIsNull()
+    AcquirerEventService.log.debug("Looking for unconsumed events")
```

Comment:
I know this isn't your review, but should this scheduled job service have its own logger?

Reply:
Probably. I'll do that separately. This is already massive, so trying to keep it as minimal as possible